src/main/java/edu/kit/hci/soli/repository/UserRepository.java

```diff
@@ -2,9 +2,8 @@
 
 import org.springframework.data.jpa.repository.JpaRepository;
 import edu.kit.hci.soli.domain.User;
-import java.io.Serializable;
 
-public interface UserRepository extends JpaRepository<User, Serializable> {
+public interface UserRepository extends JpaRepository<User, Long> {
```

Comment:
I hope this is ok?

Reply:
Yes, this is fine, prob better than Serializable anyway...